MetabolicCompass/AccountManager/Charts+Extended/CycleChartRenderer.swift

```diff
@@ -195,7 +195,7 @@ class CycleChartRender: PieChartRenderer {
                             text: valueText,
                             point: labelPoint,
                             align: align,
-                            attributes: [NSFontAttributeName: valueFont, NSForegroundColorAttributeName: dataSet.valueTextColorAt(j)]
+                            attributes: [NSAttributedStringKey.font: valueFont, NSAttributedStringKey.foregroundColor: dataSet.valueTextColorAt(j)]
```

Comment:
NSAttributedStringKey.font and others keys
can be omitted if compiler known that `attributes` should be [NSAttributeStringKey: Any]
e.g.
`attributes: [.font: valueFont, .foregroundColor: dataSet.valueTextColorAt(j)]`